learn/core_concepts/indexes.md

```diff
@@ -145,3 +145,9 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/settings.md#update-typo-tolerance-settings).
 
 [Learn more about typo tolerance.](/learn/configuration/typo_tolerance.md)
+
+## Swapping indexes
+
+Suppose you want to update your `movies` index in production because you changed the schema or settings. The changes you made are in `movies_new`. To sync `movies_new` with Meilisearch in production, you will use index swapping. This means that the schema, settings, and task history of  `movies` will be swapped with the schema, settings, and task history of `movies_new` without any downtime for the search clients. The task history of `enqueued` tasks is not modified.
```

Comment:
I would have put "Suppose you want to deploy a new index version in production **without any downtime**..." first